src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -130,6 +144,8 @@ class PdlClient(
     }
 
     companion object {
+        private val NAVN_CACHE_KEY_PREFIX = "pdl-navn-"
+        private val CACHE_EXPIRE_SECONDS = 24L * 3600
```

Comment:
Hvor ofte kommer de til å kalle denne `navn` egentlig? Når man sender forhåndsvarsel og ved journalføring? 
Så i teorien så henter vi navn ved selve opprettelsen av forhåndsvarselet og cacher resultatet, og så bruker man det cachede i journalføringen? Da sparer vi ett kall i så fall.

Kunne man feks cachet enda lenger?

Reply:
Spørsmålet går kanskje mer mot generell cache-strategi - vi kunne sikkert pre-loaded cache i flere av løsningene før veiledere kommer på jobb, slik vi gjør i oversikten med cache mot tilgangskontroll på adressebeskyttelse og skjerming 👍🏼

Reply:
Sånn det er nå så blir det ved generering av pdf og journalføring, ja, så to kall. Mulig det ikke er noe stort problem - journalføringen skjer jo dessuten asynk så der har ikke ytelsen så mye å si. Men ingen stor kostnad å legge det til heller. Samt at det blir noe mer robust mot midlertidig hikke i pdl f.eks.

Kan godt være vi kunne cachet lenger, men det er vel ikke mange varsler man skal sende på samme person, så akkurat her må vi cache lenge for at det skal ha en effekt.

Reply:
Ja sant, vanskelig å cache lenge nok sånn sett 🤔